docs/source/ci-cd.md

```diff
@@ -51,7 +51,8 @@ jobs:
             echo 'export PATH=$HOME/.rover/bin:$PATH' >> $BASH_ENV
       - checkout
       # after rover is installed, you can run it just like you would locally!
-      - run: rover graph check my-graph@prod --schema ./schema.graphql
+      # only run this command with the `--background` flag if you have the Apollo Studio GitHub integration enabled on your repository
+      - run: rover graph check my-graph@prod --schema ./schema.graphql --background
```

Comment:
I assume there's no "--check-status" kind of thing that would make sense here as well? Excuse my ignorance please; I just recall we had discussed that earlier.

Reply:
no `--check-status` is implemented in rover atm - running a check command _without_ the `--background` flag will poll continuously until the check is done or it times out, and if it times out it links to studio